main.js

```diff
@@ -4990,6 +4998,18 @@ function createRoomCallback(passAdd, passAdd2) {
 			getById("miniPerformer").innerHTML = '<button id="press2talk" onmousedown="event.preventDefault(); event.stopPropagation();" style="width:auto;margin-left:5px;height:45px;border-radius: 38px;" class="float" onclick="press2talk(true);" title="You can also enable the director`s Video Output afterwards by clicking the Setting`s button"><i class="las la-headset"></i><span data-translate="push-to-talk-enable"> enable director`s microphone or video</span></button>';
 		}
 		getById("miniPerformer").className = "";
+        
+        var tabindex = 26;
+        if(session.rooms && session.rooms.length > 0){
+            var container = getById("rooms");
+            container.innerHTML += 'Arm Transfer: ';

```

Comment:
`.innerText` is probably better to use here, even tho there is no user-generated input it is safer to use it instead of `.innerHTML`. To better understand why this doc from Mozilla might help: https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Safely_inserting_external_content_into_a_page.